paf-mvp-core-js/tests/test-cases/audit-log.test.ts

```diff
@@ -0,0 +1,103 @@
+import * as fs from 'fs';
```

Comment:
Could be interesting to add unit test for `findTransactionPath`, which is the most complex part of the code.

For example, I'm not sure this case is tested (https://github.com/prebid/paf-mvp-implementation/pull/101/files#diff-5845a944e43e07856fed8a84ea013db45573b0d5f52ea62f31e23704cbb44912R74)

```javascript
if (current.remainingChildren.length == 0) {
      current = undefined;
    }
```

Also, having unit tests around it would make it safer to refactor ;-)

Reply:
This method is already tested. Basically, this is only what we test. Check how it is called in the `buildAuditLog`.